README.md

```diff
@@ -39,6 +39,55 @@ library = foreman-ansible-modules/modules
 
 Now your playbooks and roles should have access to the modules contained in the repository for use, testing or development of new modules.
 
+## How to debug this repository
+
+Clone both ansible and this project
+
+```sh
+git clone https://github.com/ansible/ansible.git
```

Comment:
I'd push for doing this debug setup from the perspective of I am in the cloned version of this repository. And putting these artifacts in a hidden directory (and/or scripting this setup with Ansible). And if eventually this is going to be required for testing, then you'll have to script it one way or another :)